src/lib/Containers/Conversation.tsx

```diff
@@ -127,9 +128,10 @@ export class Conversation extends React.Component<Props, State> {
 
     return (
       <Composer
-        disabled={this.state.sendingMessage}
+        disabled={this.state.sendingMessage || !this.state.isConnected}
+        value={this.state.submissionError}
```

Comment:
I’m not a big fan of this name, I wonder if it could be something like `failedMessageText` ?